hoomd/mpcd/DryingDropletStreamingMethod.cc

```diff
@@ -41,6 +47,16 @@ void mpcd::DryingDropletStreamingMethod::stream(unsigned int timestep)
     // use peekStream since shouldStream will be called by parent class
     if(!peekStream(timestep)) return;
 
+    /*
+     * Validating the geometry, because the interface is shrinking, it is
+     * sufficient to validate only the first time the geometry is set.
+     */
```

Comment:
This extended comment is no longer needed since the logic here is simpler now:
```suggestion
// validate geometry
```